src/util/api-request-handler.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpClient, configHandler } from "@contentstack/cli-utilities";
+import { formatErrors } from "./error-helper";
+
+interface RequestParams {
+  orgUid: string;
+  queryParams?: Record<string, any>;
+  payload?: any;
+  url: string;
+}
+
+export async function apiRequestHandler(params: RequestParams): Promise<any> {
+  const { orgUid, queryParams, payload, url } = params;
+  const authtoken = configHandler.get('authtoken');
+  
+  const headers = {
+    organization_uid: orgUid,
+    authtoken
+  }
+  const httpClient = new HttpClient()
+  httpClient.headers(headers)
+  if (queryParams) 
+    httpClient.queryParams(queryParams);
+
+  try {
+    const response = await httpClient.put(url, payload)
+    const { status, data } = response;
```

Comment:
@harshithad0703 , Please use switch or if condition because we can't use PUT for every command.

let response
if(type === 'reinstall'){
  response = await httpClient.put(url, payload)
}else if(type === 'create'){
}

Reply:
made these changes